pkg/cluster-info/cluster-info.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster_info
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openebs/openebsctl/pkg/client"
+	"github.com/openebs/openebsctl/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/printers"
+)
+
+// ShowClusterInfo shows the openebs components and their status and versions
+func ShowClusterInfo() error {
+	k, _ := client.NewK8sClient("")
+	var clusterInfoRows []metav1.TableRow
+	for casType, componentNames := range util.CasTypeToComponentNamesMap {
+		componentDataMap, _ := getComponentDataByComponents(k, componentNames, casType)
+		if len(componentDataMap) != 0 {
+			status, working := "", ""
+			if casType == util.LocalDeviceCasType {
+				var err error
+				status, working, err = getLocalPVDeviceStatus(componentDataMap)
+				if err != nil {
+					continue
+				}
+			} else {
+				status, working = getStatus(componentDataMap)
+			}
+			version := getVersion(componentDataMap)
+			namespace := getNamespace(componentDataMap)
+			clusterInfoRows = append(
+				clusterInfoRows,
+				metav1.TableRow{Cells: []interface{}{casType, namespace, version, working, util.ColorStringOnStatus(status)}},
+			)
+		}
+	}
+	if len(clusterInfoRows) == 0 {
+		fmt.Println("None Of the OpenEBS Storage Engines are installed in this cluster")
```

Comment:
Also return an error to the caller so that error codes can be set accordingly by `cmd.Run`, after this change, the else will not be required(gosimpl will throw errors)